e2e/package.json

```diff
@@ -0,0 +1,16 @@
+{

```

Comment:
Why are you adding a different package.json only for cypress? I think that it's easier to add it to the root package.json.

Reply:
I tried to separate E2E because sometimes is easier to have in two different places to deployment with CI, but I improved the main package.json just to don't need to check in other ones

Reply:
You still need this Cypress package.json to install the dependencies and execute the commands from the main one, so, I don't see the difference.